syncthing.te

```diff
@@ -0,0 +1,90 @@
+policy_module(syncthing, 1.0.0)
+
+# Declarations
+attribute_role syncthing_roles;
+role syncthing_roles types syncthing_t;
+
+type syncthing_t;
+type syncthing_exec_t;
+# should only ever be execed non-root, but can be started by systemd --user as a service
+init_daemon_domain(syncthing_t, syncthing_exec_t)
+userdom_user_application_domain(syncthing_t, syncthing_exec_t)
+userdom_use_user_ptys(syncthing_t)
```

Comment:
use userdom_use_user_terminals(syncthing_t) instead or preferably userdom_use_inherited_user_terminals(syncthing_t) if that is available